cmake-docker-environment/labs/ninja-example/readme.md

```diff
@@ -0,0 +1,17 @@
+# ninja-example
+The project for this lesson contains the same code as cmake-example. However, instead of creating an executable with Make, this lesson will do the same work using the Ninja generator.
+
+## Exercise 1: Examine the generators found in the lab environment
+   ```
+   $ cmake --help
```

Comment:
Usually if there is a dollar sign that means its an output example. https://github.com/markdownlint/markdownlint/issues/5

Since this is copyable it would be nice if it was removed to make the readers life a touch easier

Reply:
The $ were removed in a later commit.